### PR TITLE
ENF-52 Credit Card Solution to Admin Order

### PR DESCRIPTION
**Pull request by rgabruel**

### CODE REVIEW

File: src/app/design/adminhtml/default/default/template/ebayenterprise_creditcard/form/cc.phtml [hunk 1]
```diff
@@ -40,7 +40,14 @@ $_fields = $this->getFormFields();
 	<li>
 		<label for="<?php echo $_fields['number'] ?>"><?php echo $this->__('Credit Card Number') ?> <em class="required">*</em></label>
 		<div class="input-box">
-			<input type="text" id="<?php echo $_fields['number'] ?>" name="payment[cc_number]" title="<?php echo $this->__('Credit Card Number') ?>" class="input-text validate-cc-number validate-cc-type" value="" data-encrypted-name="payment[cc_number]" />
+			<?php if (!$this->getInfoData('cc_number')) : ?>
+				<input type="text" id="<?php echo $_fields['number'] ?>" name="payment[cc_number]" title="<?php echo $this->__('Credit Card Number') ?>" class="input-text validate-cc-number validate-cc-type" value="" data-encrypted-name="payment[cc_number]" encrypted="false"/>
```
Comment (kojiromike) [inline src/app/design/adminhtml/default/default/template/ebayenterprise_creditcard/form/cc.phtml]: encrypted isn't a legit attribute – I think you'd need to use a data-* element. You could do data-client-encrypted or something

File: src/app/design/adminhtml/default/default/template/ebayenterprise_creditcard/form/cc.phtml [hunk 2]
```diff
@@ -70,7 +77,11 @@ $_fields = $this->getFormFields();
 		<label for="<?php echo $_fields['cid'] ?>"><?php echo $this->__('Card Verification Number') ?> <em class="required">*</em></label>
 		<div class="input-box">
 			<div class="v-fix">
-				<input type="text" title="<?php echo $this->__('Card Verification Number') ?>" class="input-text cvv required-entry validate-cc-cvn" id="<?php echo $_fields['cid'] ?>" name="payment[cc_cid]" value="" data-encrypted-name="payment[cc_cid]" />
+				<?php if (!$this->getInfoData('cc_cid')) : ?>
+					<input type="text" title="<?php echo $this->__('Card Verification Number') ?>" class="input-text cvv required-entry validate-cc-cvn" id="<?php echo $_fields['cid'] ?>" name="payment[cc_cid]" value="" data-encrypted-name="payment[cc_cid]" encrypted="false"/>
+				<? else: ?>
```
Comment (ryaan-anthony) [inline src/app/design/adminhtml/default/default/template/ebayenterprise_creditcard/form/cc.phtml]: I havent noticed the use of short tags, but isnt this against PSR2?

File: src/app/design/adminhtml/default/default/template/ebayenterprise_creditcard/form/cc.phtml [hunk 3]
```diff
@@ -40,7 +40,14 @@ $_fields = $this->getFormFields();
 	<li>
 		<label for="<?php echo $_fields['number'] ?>"><?php echo $this->__('Credit Card Number') ?> <em class="required">*</em></label>
 		<div class="input-box">
-			<input type="text" id="<?php echo $_fields['number'] ?>" name="payment[cc_number]" title="<?php echo $this->__('Credit Card Number') ?>" class="input-text validate-cc-number validate-cc-type" value="" data-encrypted-name="payment[cc_number]" />
+			<?php if (!$this->getInfoData('cc_number')) : ?>
+				<input type="text" id="<?php echo $_fields['number'] ?>" name="payment[cc_number]" title="<?php echo $this->__('Credit Card Number') ?>" class="input-text validate-cc-number validate-cc-type" value="" data-encrypted-name="payment[cc_number]" encrypted="false"/>
+			<? else: ?>
```
Comment (kojiromike) [inline src/app/design/adminhtml/default/default/template/ebayenterprise_creditcard/form/cc.phtml]: Use proper php tags, please.

**Comment by rgabruel** [comment]
@ryaan-anthony and @kojiromike thanks for the code review.

**Comment by ryaan-anthony** [comment]
Confirmed it's working, nice work!

**Comment by ryaan-anthony** [comment]
Spoke too soon. Got this error on submit:

`HTTP result 500 for post to https://tst05-epapi-na.gsipartners.com/v1.0/stores/MAGT2/payments/creditcard/auth/VC.xml.\n<?xml version=\"1.0\" encoding=\"UTF-8\"?>\n<Fault xmlns=\"http://api.gsicommerce.com/schema/checkout/1.0\">\n   <CreateTimestamp>2016-01-14T16:13:18.794Z</CreateTimestamp>\n   <Code>SystemProcessingError</Code>\n   <Description>A system error has occurred.</Description>\n</Fault>`

**Pull request merged**